kubernetes/resource_kubernetes_env_test.go

```diff
@@ -9,11 +9,12 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 	appsv1 "k8s.io/api/apps/v1"
+	batchv1 "k8s.io/api/batch/v1"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func TestAccKubernetesEnv_basic(t *testing.T) {
+func TestAccKubernetesEnv_ConfigMapBasic(t *testing.T) {
```

Comment:
Maybe call this one `DeploymentBasic` since the test fixture contains a deployment?